### ISSUE TITLE
runtime error in Mac mini

### ISSUE THREAD

**Issue by lhang** [gao-sun/eul#4]
### environment:
macOS: 10.15.6
hardware: Mac mini (2018)
Xcode: Version 11.5

### error log:
```
2020-09-14 13:52:14.863145+0800 eul[5549:100817] It's not legal to call -layoutSubtreeIfNeeded on a view which is already being laid out.  If you are implementing the view's -layout method, you can call -[super layout] instead. Break on void _NSDetectedLayoutRecursion(void) to debug.  This will be logged only once.  This may break in the future.
Fatal error: Double value cannot be converted to Int because it is either infinite or NaN: file /Library/Caches/com.apple.xbs/Binaries/SwiftPrebuiltSDKModules_macOS/install/TempContent/Objects/EmbeddedProjects/CrossTrain_macOS_SDK/macOS_SDK/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/lib/swift/Swift.swiftmodule/x86_64.swiftinterface, line 29400
2020-09-14 13:52:14.879331+0800 eul[5549:100817] Fatal error: Double value cannot be converted to Int because it is either infinite or NaN: file /Library/Caches/com.apple.xbs/Binaries/SwiftPrebuiltSDKModules_macOS/install/TempContent/Objects/EmbeddedProjects/CrossTrain_macOS_SDK/macOS_SDK/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/lib/swift/Swift.swiftmodule/x86_64.swiftinterface, line 29400
```

**Comment by gao-sun** [comment]
我稍后debug一下

**Comment by gao-sun** [comment]
https://github.com/gao-sun/eul/releases/latest

请尝试下载最新版看是否能解决问题

**Comment by gao-sun** [comment]
@lhang :
@missdeer helped me to find out the root of this issue and it has been resolved in 1.0.6. would you mind to download the new version and try again? thanks

**Comment by lhang** [comment]
> @lhang :
> @missdeer helped me to find out the root of this issue and it has been resolved in 1.0.6. would you mind to download the new version and try again? thanks

it works! 👍
